### PR TITLE
Exclusive Display Benchmark

### PR DESCRIPTION
**Pull request by sohamm17**
This MR compares exclusive display vs default mode. It also provides performance of both modes in presence of other simultaneous headless applications.

### CODE REVIEW

File: benchmarks/exclusive_display/README.md [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+
+### Maximum End-to-end Latency
+
+The maximum end-to-end latency results are given below:
+
+![Maximum Latency](exclusive_max_latency.png)
+
+In the above graph, the maximum end-to-end latency for the default mode increases from 15 ms to 23
+ms when the number of headless applications rises from 0 to 3. For more than 3 headless
+applications, the maximum end-to-end latency in default mode is more than 200 ms.
```
Comment (AndreasHeumann) [inline benchmarks/exclusive_display/README.md:53]: Isn't `default` mode using the X11 compositor? The graph is comparing `default` to `exclusive`, but this paragraph is `default` (or `headless`?) vs. `exclusive`.

File: benchmarks/exclusive_display/README.md [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+
+## Results
+
+For the experiment, we use the endoscopy tool tracking application which is using the display
+monitor for outputs in two modes, as said above. This application is executed with `realtime: false`
+for the
+video stream replayer source, so that the source feeds the frames as fast as possible, without an
+external frame-rate limitation.
+
+For the headless applications, we run the same endoscopy
```
Comment (AndreasHeumann) [inline benchmarks/exclusive_display/README.md:37]: Introduction is about `exclusive` mode, but here it's talking about `headless`.

**Review by AndreasHeumann — COMMENTED** [review]
Nice work. I'm a bit confused about `exclusive` vs. `headless` vs `default` mode. Text and graphs don't seem to match.

File: benchmarks/exclusive_display/README.md (hunk 2 above)
Comment (sohamm17) [reply benchmarks/exclusive_display/README.md:37]: @AndreasHeumann The primary application whose performance is benchmarked is either in "exclusive" mode or "default" mode. The graphs show the performance of this single primary application.
We run a varying number of "headless" applications in the background. The number of these background "headless" applications vary from 0 to 11.
This is clarified in the second paragraph of the Introduction section above.

File: benchmarks/exclusive_display/README.md (hunk 1 above)
Comment (sohamm17) [reply benchmarks/exclusive_display/README.md:53]: Sorry for the confusion. "headless" applications are background applications. We compare the performance of "exclusive" and "default" mode of a single primary endoscopy tool tracking application.

**Review by AndreasHeumann — APPROVED** [review]

**Pull request merged**